projects/angular/src/forms/datepicker/_datepicker.clarity.scss

```
@@ -131,8 +132,10 @@
 
     &.has-action-buttons {
       height: calc(
-        datepicker-variables.$clr-calendar-height + datepicker-variables.$clr-calendar-action-buttons-height
+        datepicker-variables.$clr-calendar-height + datepicker-variables.$clr-calendar-action-buttons-height +
+          calc(2 * datepicker-variables.$clr-calendar-button-gap)
       );
+      row-gap: calc(2 * datepicker-variables.$clr-calendar-button-gap);
```

Whys `height` and `row-gap` are based on `$clr-calendar-button-gap`? Make sense to be regular token.
```suggestion
        datepicker-variables.$clr-calendar-height + datepicker-variables.$clr-calendar-action-buttons-height +
          tokens.$cds-global-space-5
      );
      row-gap: tokens.$cds-global-space-5;
```

---

Fixed in [948b80d](https://github.com/vmware-clarity/ng-clarity/pull/1706/commits/948b80d3d202cba561298fdb5a6636cfd8e84a57)